appender.go

```diff
@@ -148,9 +148,15 @@ func TestAppender(t *testing.T, store types.Storager) {
 
 			size := rand.Int63n(4 * 1024 * 1024) // Max file size is 4MB
 			content, _ := ioutil.ReadAll(io.LimitReader(randbytes.NewRand(), size))
-			r := bytes.NewReader(content)
 
-			_, err = ap.WriteAppend(o, r, size)
+			r := bytes.NewReader(content[:size/2])
```

Comment:
How about using the `content` twice instead of split them into halves? (which introduces more logic)

Reply:
sounds good.